UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -175,8 +176,6 @@ void IInitialise.Initialise()
 	{
 		// Set up server defaults, needs to be loaded here to ensure gameConfigManager is load.
 		LoadConfig();
-		RespawnCurrentlyAllowed = RespawnAllowed;
-		NextGameMode = InitialGameMode;
```

Comment:
Was this not needed?

Reply:
I need to test this for a sec, GameMode stuff worked fine last time I tested, so I don't remember why I deleted this.